learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -27,7 +27,7 @@ MeiliSearch is intended to deliver performant instant search experiences aimed a
 Elasticsearch can sometimes be too slow if you want to provide a full instant search experience. Most of the time, it is significantly slower in returning search results compared to MeiliSearch.
 MeiliSearch is a perfect choice if you need a simple and easy tool to deploy a typo-tolerant search bar that provides a prefix searching capability, makes search intuitive for users, and returns them their results instantly with near-perfect relevance.
 
-### MeiliSearch vs. Algolia
+### MeiliSearch vs algolia
```

Comment:
Algolia is a proper noun (we are speaking about a company called Algolia).

Suggestion: `MeiliSearch vs Algolia`